src/components/friends/FriendCard.tsx

```diff
@@ -9,7 +9,7 @@ import { TitleSection, WrapperSection, Text } from '../../styles/GlobalStyle.css
 import { color, padding, margin, fontSize } from '../../styles/theme'
 import { SecondaryButton } from '../buttons/SecondaryButton'
 import { ResponsiveImage } from '../images/ResponsiveImage'
-import { friendUrl } from './friends-url-service'
+import { buildFriendUrl } from './friends-url-service'
```

Comment:
może to trzymać po prostu w pliku friend-service.ts? razem z innymi funkcjami dla friends?